### DISCUSSION TITLE
achievements/svrp2020/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme#102] [General]
# achievements/svrp2020/

Developed by Association of Information Security Professionals (AiSP) and Cyber Security Agency of Singapore (CSA), the SVRP aims to encourage volunteerism amon

https://www-conanzhang-tech.pages.dev/achievements/svrp2020/

**Comment by conanzhangtech** [comment]
Test

**Reply by razonyang** [reply]
Hi @conanzhangtech, thanks for using this theme.

To get notified when others comment, you need to tweak the `giscus` parameters (`params.yml`). See also.

- For blog layout: https://hbstack.dev/en/modules/blog/comments/giscus/
- For docs layout: https://hbstack.dev/en/modules/docs/comments/giscus/
